docs/monoDrive_home/Weather.md

```diff
@@ -71,139 +54,167 @@ window for the Actor. Scroll down to the "Weather Description" in the "Details"
 window to find the weather settings pictured below.
 
 <p class="img_container">
-  <img class="lg_img" src="../img/details_weather_actor.png" />
+  <img class="lg_img" src="../img/weather_ui.png" />
 </p>
 
```

Comment:
For consistency with the rest of the documentation, add " - " for bullet points in front of configuration values in a list format like this.